### DISCUSSION TITLE
What is your favorite animal?

### DISCUSSION THREAD

**Discussion by Maro451** [NAIST-SE/AttractingContributors#1153] [2024 Attraction Contributions]
https://github.com/Maro451/Animal

**Comment by ryosuke-sota** [comment]
I made a PR on your repo.
Could you contribute to mine?
Here : https://github.com/ryosuke-sota/AttractingContributors.git
Thank you.

**Comment by 3207-Rhims** [comment]
https://github.com/3207-Rhims/AttractingContributors-Movie , would you be kind enough to contribute to my project? I made a PR for you.

**Comment by kohanaist** [comment]
I made a PR on your repo.
Could you contribute to mine?
https://github.com/kohanaist/AttractingContributors-technical-or-specialized-book

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI
Thank you very much！

**Comment by wangzaixiaoliunai** [comment]
I have created a PR.
Could you please contribute to mine?
https://github.com/wangzaixiaoliunai/AttractingContributors-musical_instruments.git
